spec/unit/provider/package/pkgng_spec.rb

```diff
@@ -169,6 +169,18 @@ def run_in_catalog(*resources)
       bash_comp_latest_version = described_class.get_latest_version('shells/bash-completion', version_list)
       expect(bash_comp_latest_version).to eq('2.1_3')
     end
+
+    it "should return nil when the package is orphaned" do
+      version_list = File.read(my_fixture('pkg.version'))
+      nmap_latest_version = described_class.get_latest_version('sysutils/orphan', version_list)
+      expect(nmap_latest_version).to be_nil
+    end
+
+    it "should return nil when the package is broken" do
+      version_list = File.read(my_fixture('pkg.version'))
+      nmap_latest_version = described_class.get_latest_version('sysutils/broken', version_list)
```

Comment:
nit: this variable still has `nmap` in the title, maybe change to something like `broken_latest_version`?

Reply:
the dreaded copy/paste mistake, fixed!